rviz_visual_testing_framework/src/visual_test_fixture.cpp

```diff
@@ -51,13 +51,14 @@ void VisualTestFixture::SetUpTestCase()
   visualizer_app_->setApp(qapp_);
 
   visualizer_app_->init(argc, argv);
+
   if (VisualTest::generateReferenceImages()) {
     visualizer_app_->loadConfig(QDir::toNativeSeparators(
-        QString::fromStdString(std::string(_SRC_DIR_PATH) + "/config/visual_tests_default_config.rviz")));
+        QString::fromStdString(src_directory_path_ + "/visual_tests_default_config.rviz")));
```

Comment:
The Path is wrong: `src_directory_path_`, as defined in the fixture constructor points just to the `share` folder, the `config` folder is missing. It should be `share_directory_path_/config/visual_tests_default_config.rviz`

Reply:
Updated to include config path